content/en/observability_pipelines/setup/datadog.md

```diff
@@ -250,7 +250,7 @@ Set up the Worker module in your existing Terraform using this sample configurat
 
 ```
 module "opw" {
-    source     = "https://github.com/DataDog/opw-terraform//aws"
+    source     = "git::https://github.com/DataDog/opw-terraform//aws"
```

Comment:
@silversupreme I approved, but just noticed that there are two `/` before `aws`, is that correct?